examples/run_lm_finetuning.py

```diff
@@ -493,9 +509,6 @@ def main():
     parser.add_argument(
         "--evaluate_during_training", action="store_true", help="Run evaluation during training at each logging step."
     )
-    parser.add_argument(
-        "--do_lower_case", action="store_true", help="Set this flag if you are using an uncased model."
-    )
```

Comment:
At this point this should be an option of the tokenizer config, so I don't think we should leave that here. (Otherwise, all tokenizer options should also be there)